Makefile

```diff
@@ -28,7 +28,6 @@ integration-test-py3: build-py3
 	docker run -v `$(HOST_TMPDIR)`:/tmp -v /var/run/docker.sock:/var/run/docker.sock docker-py3 py.test -rxs tests/integration_test.py
 
 integration-dind: build build-py3
-	docker build -t dpy-tests -f ./tests/Dockerfile .
```

Comment:
Is this being done as part of the jenkins config? Maybe just change this to a comment so that users can run it locally ?